packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,7 +1,12 @@
-exports.onRouteUpdate = function(location) {
+var lastPath = null
+exports.onRouteUpdate = function(newRoute) {
   // Don't track while developing.
   if (process.env.NODE_ENV === `production`) {
-    ga(`set`, `page`, location.pathname)
-    ga(`send`, `pageview`)
+    var newPath = newRoute.pathname || newRoute.location.pathname
```

Comment:
@KyleAMathews this `newPath` `lastPath` thing can be solved somewhere else probably, if you can point me in the right direction!

Essentially, as we navigate through a site, the `onRouteUpdate` function gets called one additional time, like:

Home page - fires once
... user goes to about page - fires twice
... user goes back to homepage - fires three times.